web/styling/box-model/borders.md

```diff
@@ -26,7 +26,9 @@ revisionQuestion:
 
 ## Content
 
-Borders occupy the space between margin and padding, providing an outline of the element. For a `border` to be displayed, it requires 3 properties: `width`, `style` and `color`. Those properties can be declared individually:
+Borders occupy the space between margin and padding, providing an outline of the element.
+
+The `border` has 3 properties: `width`, `style`, and `color`. Each of them can be declared individually:
```

Comment:
hmm, let's leave out the "the" at the start; we could say "The `border` property" but then we'd repeat ourselves "The `border` property has three properties"

```suggestion
`border` has three properties: `width`, `style`, and `color`. Each of them can be declared individually:
```